src/components/SafeListSidebar/SafeList/SafeListItem.tsx

```diff
@@ -59,14 +58,14 @@ const SafeListItem = ({
     onSafeClick()
   }
 
+  const routeSlugs: SafeRouteParams = {
```

Comment:
I always thought a slug (as popularized by WordPress) was referring to the whole pathname. This is more like routeParams.

Reply:
I know what you mean, but React Router defines them as slugs. I'll rename them to params though.